datadog_lambda/tracing.py

```diff
@@ -689,10 +874,20 @@ def create_inferred_span_from_http_api_event(event, context):
         "span_type": "http",
     }
     tracer.set_tags({"_dd.origin": "lambda"})
+    inferred_span_start_ns = None
+    if decode_authorizer_context:
+        injected_authorizer_data = get_injected_authorizer_data(
+            event, _EventSource(EventTypes.API_GATEWAY, EventSubtypes.HTTP_API)
+        )
+        if injected_authorizer_data:
+            inferred_span_start_ns = int(
+                injected_authorizer_data.get(Headers.Parent_Span_Finish_Time)
+            )
+    inferred_span_start_ns = inferred_span_start_ns or request_time_epoch_ms * 1e6
```

Comment:
we're using 1e6 in a couple of places in this PR, although I know we're converting ms to ns, and your variables are named perfectly, it might be helpful to extract this into a constant? `MILLIS_TO_NANOS` maybe? There probably is one in the time package.

Reply:
I tried to first use a const `MILLIS_TO_NANOS = 1e6`. I didn't like it because it lacks the operator. It should be more like `MILLIS_TO_NANOS(x) :  1e6 * x`.
Then I tried to define a function `millis_to_nano(x)`. But i feel using a function is a bit overkill and tends to hide information.
I think I'll keep it explicit like this for now.